x/wasm/internal/keeper/handler_plugin_test.go

```diff
@@ -276,6 +288,27 @@ func TestEncoding(t *testing.T) {
 				},
 			},
 		},
+		// TODO: alpe? can you add an example with sub-interfaces (where the UnpackInterfaces call would be needed)
```

Comment:
I am not sure how to create such a case. (I also think we only have bank, staking, and distribution registered here in the tests).

If you could add such a test to trigger this that would be great. Or if it is a pain, please open up an issue on it and explain more or less what is needed. I can put that down lower on the priority list

Reply:
I found `Any` types in sdk (master). 
* MsgGrantAuthorizationRequest
* MsgGrantFeeAllowance
* MsgSubmitProposal
* MsgCreateValidator
I guess `MsgSubmitProposal` would be a good candidate. I can add a test.